<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-ONQP60F" ignored="0" name="Default test(failed)" tests="4" failures="1" timestamp="2021-10-12T15:26:19 IST" time="24.694" errors="0">
  <testcase name="CreateOrgTest" time="0.012" classname="practice.CreateOrgTest">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <testcase name="CreateOrgTest" time="0.002" classname="practice.CreateOrgTest">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <testcase name="CreateOrgTest" time="0.001" classname="practice.CreateOrgTest">
    <skipped/>
  </testcase> <!-- CreateOrgTest -->
  <testcase name="CreateOrgTest" time="0.001" classname="practice.CreateOrgTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at practice.CreateOrgTest.CreateOrgTest(CreateOrgTest.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- CreateOrgTest -->
</testsuite> <!-- Default test(failed) -->
